examples/llm_text_to_text/simple_model_training.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+
+# # Simple Model Training Example
+#
+# This is a simple example of how to use the LLM model type to train
+# a model on a simple question and answer dataset. It uses the
+# facebook/opt-350m model as the base LLM model.
+
+# Import required libraries
+import logging
+import shutil
+
+import pandas as pd
+import yaml
+
+from ludwig.api import LudwigModel
+
+# clean out prior results
+shutil.rmtree("./results", ignore_errors=True)
+
+qa_pairs = {
```

Comment:
small nit. here and for the next examples I would change the datasets to be passed to pandas to lists ofr dictionaries instead of dictionaries of lists

Reply:
Updated